org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -83,6 +83,33 @@ class Translator {
     }
 
     /**
+     * Subscribe to notifications for a platform.
+     * This function is intended to be called by the platform translator for initial subscription,
+     * and on the hub translator (this) for verification.
+     */
+    postSubscribe(subscriptionInfo) {
+        return this._hasValidEndpoint().then((isValid) => {
+            if (isValid == false) return undefined;
+
+            var requestPath = '/subscription/' + subscriptionInfo.controlId;
+            return this._makeRequest(requestPath, 'POST', '');
```

Comment:
I don't know, this code is only being moved (from below _subscribe(...)

---
In reply to: [92023354](https://github.com/openT2T/translators/pull/136#discussion_r92023354) [](ancestors = 92023354)